.github/workflows/publish.yml

```diff
@@ -56,3 +56,9 @@ jobs:
           node_modules/.bin/vsce publish --pre-release --packagePath vscode-ruby-lsp.vsix
         env:
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
+
+      # Stable releases for OpenVSX
+      - name: Publish extension on OpenVSX
+        if: "!github.event.release.prerelease"
+        run: |
+          npx ovsx publish -p ${{ secrets.OPENVSX_TOKEN }} --yarn
```

Comment:
Ah, without a filename it would build from source:
https://github.com/eclipse/openvsx/wiki/Publishing-Extensions#5-package-and-upload

I've changed it to supply a filename to make it consistent.